version_control/katalyst_by_codurance/css/templates/home.css

```
@@ -147,6 +145,30 @@
     margin-inline: auto;
 }
 
-.section-header {
+/* Contact section */
+
+.contact {
+    padding: var(--space-2xl) 0;
+    background: var(--light-orange) url("https://www.katalystbycodurance.com/hubfs/KATALYST%20BY%20CODURANCE/Microsite/bg-logo-isotype-contact-us.svg") bottom right no-repeat;
+}
+
+.contact .section-header {
+    max-width: 37.5rem;
```

When implementing this section, I've started to use characters as a max width for the section rather a rem value. I believe it's easier to remember a full number or we can extract this width to a variable
```suggestion
    max-width: var(--section-max-width);
```